sdk/core/Azure.Core.TestFramework/src/ClientTestFixtureAttribute.cs

```diff
@@ -15,13 +16,49 @@ namespace Azure.Core.TestFramework
 {
     public class ClientTestFixtureAttribute : NUnitAttribute, IFixtureBuilder2, IPreFilter
     {
+        private readonly struct TestVersion : IComparable
+        {
+            public IComparable Comparable { get; }
```

Comment:
Do we need this property? Can't we just store the converted data in `Object` and use CompareTo on that.

Reply:
Object is actually the only thing needed outside this struct on line 169, but I can make this a private readonly field